test/integration/gardenlet/managedseed/managedseed_suite_test.go

```diff
@@ -204,6 +204,8 @@ var _ = BeforeSuite(func() {
 				WaitSyncPeriod:   &metav1.Duration{Duration: 5 * time.Millisecond},
 				ConcurrentSyncs:  pointer.Int(5),
 				SyncJitterPeriod: &metav1.Duration{Duration: 50 * time.Millisecond},
+				// This controller is pretty heavy-weight, so use a higher duration.
+				SyncPeriod: &metav1.Duration{Duration: time.Minute},
```

Comment:
Why is this suddenly needed?

Reply:
same reason as in https://github.com/gardener/gardener/pull/7248#discussion_r1066692943